client/reader/site-stream/index.jsx

```diff
@@ -127,10 +128,6 @@ const SiteStream = React.createClass( {
 			title = this.translate( 'Loading Site' );
```

Comment:
Well, the whole thing seems to be slightly broken. There's one other controller that uses `SiteStream`, `blogListings`, which is in charge of `/read/blogs/:blog_id`. Try `https://wordpress.com/read/blogs/developer.wordpress.com` (i.e. production :) -- title remains at `Loading Site` state forever...

cc @blowery @bluefuton